src/plot_api/plot_api.js

```diff
@@ -404,6 +404,11 @@ function opaqueSetBackground(gd, bgColor) {
     setBackground(gd, bgColor);
 }
 
+function blendSetBackground(gd, bgColor) {
+    var blend = Color.combine(bgColor, 'white');
```

Comment:
RE `'opaque'` vs `'blend'`, they don't generate the same PNG (`gm` compare generates a diff) but I can't find a case where the diff is detectable to my :eyes:

So, I guess a can make the new image server use `'opaque'` instead of `'blend'` and :hocho: `blendSetBackground`.